### PR TITLE
IE and Edge issues - Failing Tests

### PR DESCRIPTION
**Pull request by jereckley**
Hey! First time trying to contribute. In short the issues that are not covered by @mattdsteele here: https://github.com/ionic-team/stencil/pull/1501 : 1) CSS variables are not working as expected in IE 2) Nothing is working in Edge. After adding a polyfill for getRootNode I can get Edge working for the most part. There are still about a dozen tests failing in edge after the poly fill is added.

### CODE REVIEW

File: test/karma/karma.config.js [hunk 1]
```diff
@@ -46,6 +46,12 @@ const localLaunchers = {
 			'--remote-debugging-port=9333'
 		]
   },
+  IEGeneral: {
+    base: 'IE'
+  },
+  EdgeGeneral: {
+    base:'Edge'
```
Comment (jereckley) [inline test/karma/karma.config.js]: Adding test support for edge and ie

**Comment by simonhaenisch** [comment]
I think in the meantime there's been lots of fixes related to IE/Edge... can you check which of the cases are still not working?

**Comment by jereckley** [comment]
I checked everything out yesterday and it was like a 100 time better in our projects. Thanks! There are still some issues but I think I we need to check our bases before raising any concerns.
